packages/ui/src/components/va-modal/VaModal.vue

```diff
@@ -256,6 +264,19 @@ export default defineComponent({
     const window = useWindow()
     const document = useDocument()
 
+    watch(valueComputed, (newValue) => {
+      // put this into watchEffect -> max recursion stack
```

Comment:
```suggestion
```

Yeah, because we don't need to change and reactive variables at the same time in watchEffect.

Reply:
Fixed